frontend/src/i18n/locales/features/login/en-sg.ts

```diff
@@ -3,6 +3,15 @@ import { Login } from '.'
 export const enSG: Login = {
   LoginPage: {
     slogan: 'Build secure government forms in minutes',
+    banner: 'You can now collect payments directly on your form!',
+    expiredSgIdSession:
+      'Your sgID login session has expired. Please login again.',
+  },
+  SelectProfilePage: {
+    accountSelection: 'Choose an account to continue to FormSG',
+    manualLogin: 'Or, login manually using email and OTP',
+    invalidWorkEmail:
+      'It may be available only to select agencies or authorised individuals. If you believe you should have access to this service, please <2><0>contact us</0></2>',
```

Comment:
Hey @SeanChinJunKai, had a second look and thought a little more on this.

I understand that this makes it really useful to reduce the number of translation keys, but I would prefer to be explicit with the translation rather than introducing a coupling with the structure of the React DOM.